packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -3594,6 +3594,25 @@ void main() {
     skip: kIsWeb, // [intended] Web uses its native context menu.
   );
 
+  // Regression test for https://github.com/flutter/flutter/issues/148934
```

Comment:
#121053 seems to be the original issue. I think we should probably close #148934 as a duplicate.